uaclient/cli.py

```diff
@@ -96,6 +96,7 @@ def attach_parser(parser=None):
     parser._optionals.title = "Flags"
     parser.add_argument(
         "token",
+        nargs="?",  # required arg custom handling in action_attach
```

Comment:
"required arg custom handling" isn't super-clear to me; I doubt I will remember what it means in a few months. :-)